contracts/alphabet/contract.go

```diff
@@ -68,7 +68,7 @@ func _deploy(data any, isUpdate bool) {
 
 // Update method updates contract source code and manifest. It can be invoked
 // only by committee.
-func Update(nefFile []byte, manifest []byte, data any) {
+func Update(nefFile, manifest []byte, data any) {
```

Comment:
I don't like this style since it breaks simple "name type" sequence, you have to walk past the comma to figure out what's going on. I can live with it though.